system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -26,19 +26,47 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
+;; Helper constants and utility functions
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(def var1-name "variable1")
+(def var1-name-new "variable1-new")
+(def del-var-name "deleted-variable")
+(def user-name "user1")
+
+(defn- get-tokens
```

Comment:
Almost -- the delete needs the token, but everywhere else I've removed it. Thanks for noticing that!